js/common/tags.go

```diff
@@ -0,0 +1,79 @@
+package common
```

Comment:
```suggestion
package tagging
```
I think we could create a dedicated package `tags` or `tagging` for it, so we could use like `tags.ApplyUserDefined`

Reply:
hmm :thinking: maybe, though this is something very specific for the JS packages and it's just a couple of helper functions :thinking: calling it `tags` or `tagging` might occupy a valuable package name in the future :sweat_smile:

Reply:
Maybe than move it to the `common` package?

Reply:
It already is in the `common` package? :confused:

Reply:
Ping @mstoykov - can you please explain what you meant here?

Reply:
Oh ... I meant that I am terrible at reading code before i finish my second :coffee: ... and was of the impression that it is in folder `tags` :facepalm: 

tl;dr-ignore me